src/helpers/zapper.js

```diff
@@ -14,14 +14,15 @@ export async function fetchTreasuryBalance() {
       `${zapperApiUrl}/balances?addresses%5B%5D=${treasury}&api_key=${zapperApiKey}`,
       {
         onmessage({ event, data }) {
-          if (event === 'totals') {
-            const { netTotal } = JSON.parse(data);
-            treasuryBalance = parseFloat(netTotal);
+          if (event === 'balance') {
+            const { app } = JSON.parse(data);
+            if (app && app.meta) {
+              treasuryBalance += parseFloat(app.meta.total);
+            }
```

Comment:
@gbonumore - might be worth throwing an error if the event doesn't match the expected API specification. 

So something like:

```js
          if (event === 'balance') {
            const { app } = JSON.parse(data);
            if (app && app.meta) {
              treasuryBalance += parseFloat(app.meta.total);
            } else {
                throw Error('Zapper API Error');
            }
     } else {
        throw Error('Zapper API Error');
}
```

From what I can see - if we don't record an expected event, `treasuryBalance` will return `undefined` - this might then cause some cascading issues.

Reply:
treasury balance returns 0 in the case balance events have no data, throwing an error for every different event results on always throwing errors, since you get some events like "end" or potentially new events that you don't need to handle.

Reply:
Ah I see - makes sense.